editor/editor_node.cpp

```diff
@@ -5110,8 +5115,15 @@ void EditorNode::_load_docks_from_config(Ref<ConfigFile> p_layout, const String
 	}
 
 	// Restore collapsed state of FileSystemDock.
+	PackedStringArray uncollapsed_tis;
 	if (p_layout->has_section_key(p_section, "dock_filesystem_uncollapsed_paths")) {
-		PackedStringArray uncollapsed_tis = p_layout->get_value(p_section, "dock_filesystem_uncollapsed_paths");
+		uncollapsed_tis = p_layout->get_value(p_section, "dock_filesystem_uncollapsed_paths");
+	} else {
+		uncollapsed_tis = { "res://" };
```

Comment:
This isn't really necessary, but it provides compatibility with pre-4.1 layout files.